frontend/src/metabase/admin/settings/components/widgets/EmbeddingCustomizationInfo.tsx

```diff
@@ -6,9 +6,9 @@ import SettingHeader from "../SettingHeader";
 
 export const EmbeddingCustomizationInfo = () => {
   const setting = {
-    description: jt`In order to remove the Metabase logo from embeds, you can always upgrade to one of our ${(
+    description: jt`In order to remove the Metabase logo from embeds, you can always upgrade to ${(
       <ExternalLink href={MetabaseSettings.upgradeUrl()}>
-        {t`paid plans.`}
+        {t`one of our paid plans.`}
```

Comment:
[See the Slack discussion for context.](https://metaboat.slack.com/archives/C03E8MQJZBM/p1657108198925169?thread_ts=1657093586.836219&cid=C03E8MQJZBM)